### PR TITLE
EM-2243: Add charts for em and dg into ithc

### PR DESCRIPTION
**Pull request by tomxelliott**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: k8s/ithc/common/dg/dg-docassembly.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: dg-docassembly
+    version: 1.0.1
+  values:
+    docassembly-api:
+      replicas: 2
+      applicationInsightsInstrumentKey: "039faf8c-4682-432e-8fc5-f9735bb4b962"
```
Comment (luigibk) [inline k8s/ithc/common/dg/dg-docassembly.yaml]: this should come from your vault ... and it doesn't look like the right key either

File: k8s/ithc/common/dg/dg-docassembly.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        DOCMOSIS_TEMPLATES_ENDPOINT: "https://docmosis.platform.hmcts.net"
+        DOCMOSIS_ENDPOINT: "https://docmosis.platform.hmcts.net/rs/render"
+        S2S_BASE_URI: "http://rpe-service-auth-provider-ithc.service.core-compute-ithc.internal"
+        DM_STORE_APP_URL: "http://dm-store-ithc.service.core-compute-ithc.internal"
+        ENABLE_FORM_DEFINITION_ENDPOINT: "false"
+        ENABLE_TEMPLATE_RENDITION_ENDPOINT: "true"
+        IDAM_API_BASE_URI: "https://idam-api.platform.hmcts.net"
+        JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
+          -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -javaagent:/opt/app/applicationinsights-agent-2.3.1.jar
+      ingressHost: dg-docassembly-ithc.service.core-compute-ithc.internal
```
Comment (luigibk) [inline k8s/ithc/common/dg/dg-docassembly.yaml]: you don't need this if you are using a recent chart java (which you should)

File: k8s/ithc/common/em/em-anno.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  values:
+    anno:
+      replicas: 2
+      applicationInsightsInstrumentKey: "039faf8c-4682-432e-8fc5-f9735bb4b962"
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/em/anno:prod-d8ae6a32
+      ingressHost: em-anno-ithc.service.core-compute-ithc.internal
+      environment:
+        S2S_BASE_URI: "http://rpe-service-auth-provider-ithc.service.core-compute-ithc.internal"
+        IDAM_API_BASE_URI: "https://idam-api.platform.hmcts.net"
```
Comment (luigibk) [inline k8s/ithc/common/em/em-anno.yaml]: All comments from above valid here as well.

File: k8s/ithc/common/dg/dg-docassembly.yaml (hunk 1 above)
Comment (tomxelliott) [reply k8s/ithc/common/dg/dg-docassembly.yaml]: I have requested this key as I am unable to access on Azure

**Pull request merged**